contracts/truefi/TrueFiPool.sol

```diff
@@ -163,38 +157,6 @@ contract TrueFiPool is ITrueFiPool, ERC20, ReentrancyGuard, Ownable {
      */
     event JoiningPauseStatusChanged(bool isJoiningPaused);
 
-    /**
-     * @dev Initialize pool
-     * @param __curvePool curve pool address
-     * @param __curveGauge curve gauge address
-     * @param __currencyToken curve pool underlying token
-     * @param __lender TrueLender address
-     * @param __uniRouter Uniswap router
-     */
-    function initialize(
```

Comment:
Should we really remove the initializer?

Reply:
Why not?